        <html><head></head><body>
        <h1>Report for cddb-py on toolchain
            <a href="index.html">android</a>
        </h1>
        <p>Return code: 5</p>
        <p>Time to run: 00:00:36</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_android -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data cddb-py
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 2041 packages in cache
===> sudo -u tuscan makepkg --nobuild --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: cddb-py 1.4-7 (Fri Nov 13 23:55:02 UTC 2015)
==> Checking runtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Packages (2) sqlite-3.9.2-1  python2-2.7.10-2

Total Installed Size:  75.75 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
checking available disk space...
installing sqlite...
installing python2...
Optional dependencies for python2
    tk: for IDLE
    python2-setuptools
    python2-pip
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading CDDB-1.4.tar.gz...
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
  -> Extracting CDDB-1.4.tar.gz with bsdtar
==> Sources are ready.
Copied source directory to /sources/cddb-py
===> sudo -u tuscan PATH=/tmp/toolchain/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin CC=arm-linux-androideabi-gcc CXX=arm-linux-androideabi-g++ SYSROOT=/build/android-ndk-r10e/platforms/android-21/arch-arm makepkg --noextract --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: cddb-py 1.4-7 (Fri Nov 13 23:55:11 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Entering fakeroot environment...
==> Starting package()...
running install
running build
running build_py
creating build
creating build/lib.linux-x86_64-2.7
copying CDDB.py -> build/lib.linux-x86_64-2.7
copying DiscID.py -> build/lib.linux-x86_64-2.7
running build_ext
building 'cdrom' extension
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/unix
arm-linux-androideabi-gcc -DNDEBUG -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/tmp/toolchain/include/python2.7 -c unix/cdrommodule.c -o build/temp.linux-x86_64-2.7/unix/cdrommodule.o
In file included from /tmp/toolchain/include/python2.7/Python.h:58:0,
                 from unix/cdrommodule.c:24:
/tmp/toolchain/include/python2.7/pyport.h:1029:2: error: #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."
 #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."
  ^
error: command 'arm-linux-androideabi-gcc' failed with exit status 1
==> ERROR: A failure occurred in package().
    Aborting...
===> Printing config.logs
===> Finished printing config.logs></pre></code>
        </body></html>
